docs/sources/datasources/tempo/_index.md

```diff
@@ -60,13 +60,13 @@ The **Trace to logs** setting configures the [trace to logs feature]({{< relref
 1. Select the target data source.
 2. Select which tags to use in the logs query. The tags you configure must be present in the spans attributes or resources for a trace to logs span link to appear.
 
-- **Single tag**
-  - Configuring `job` as a tag and clicking on a span link will take you to your configured logs datasource with the query `{job='value from clicked span'}`.
-- **Multiple tags**
-  - If multiple tags are used they will be concatenated so the logs query would look like `{job='value from clicked span', service='value from clicked span'}`.
-- **Mapped tags**
-  - For a mapped tag `service.name` with value `service`, clicking on a span link will take you to your configured logs datasource with the query `{service='value from clicked span'}` instead of `{service.name='value from clicked span'}`.
-  - This is useful for instances where your tracing datasource tags and your logs datasource tags don't match 1:1.
+   - **Single tag**
+   - Configuring `job` as a tag and clicking on a span link will take you to your configured logs datasource with the query `{job='value from clicked span'}`.
```

Comment:
@chri2547 this line should prob be indented more right?

Reply:
Good catch, thank you.